index.js

```diff
@@ -6,11 +6,11 @@ import './adhocracy4/static/global_jquery'
 export { default as alert } from './adhocracy4/static/Alert'
 export { default as api } from './adhocracy4/static/api'
 export { default as comments } from './adhocracy4/comments/static/comments/react_comments'
-export * as commentsAsync from './adhocracy4/comments_async/static/comments_async'
+export * as commentsAsync from './adhocracy4/comments_async/static/comments_async/react_comments'
```

Comment:
so we don't double up names I would call this react_comments_async or something?

Reply:
done